apb.yml

```diff
@@ -11,6 +11,11 @@ metadata:
   longDescription: "An APB deploying the AeroGear UnifiedPush Server with persistent storage and authentication through Openshift"
   imageUrl: "https://pbs.twimg.com/profile_images/1794440005/aerogear_icon-1_400x400.png"
   documentationUrl: "https://github.com/aerogearcatalog/unifiedpush-apb/blob/master/docs/modules/ups_openshift/index.asciidoc"
+  sdkDocumentation:
```

Comment:
these can't be parsed in the broker in this format. may need to change to be something like 
```
- android-sdk-docs
- ios-sdk-docs
```